CSharpLinq/CSharpLinq/Program.cs

```diff
@@ -0,0 +1,99 @@
+﻿using CSharpOOP2;
+using System.Diagnostics;
+#region Task_1. List of numbers divisible by three
+//List<int> numbersToDivide = new() { 32, 50, 40, 77, 36, 93, 57, 4, 37, 7 };
+
+//var numbersDivisibleByThree = from r in numbersToDivide
+//                              where r % 3 == 0
+//                              select r;
+
+//Console.WriteLine($"The list of numbers divisible by 3:");
+
+//foreach (var number in numbersDivisibleByThree) { Console.WriteLine(number); }
+#endregion
+#region Task_2. List of positive numbers less than 10 and negative numbers
+
+//List<int> randomNumbers = new() { -7, 7, 14, -11, 8, 11, -14, 10, -12, -8 };
+
+//var SelectedNumbers = randomNumbers.Where(p => p < 10).Select(r => r).ToList();
+
+//Console.WriteLine($"List of positive numbers, that are less than 10 and negative numbers:");
+
+//foreach (var r in SelectedNumbers) { Console.WriteLine(r); }
+#endregion
+#region Task_3. Sorted list of names with an even number of characters
+//List<string> randomNames = new() {"Malika", "Enya", "Cruz", "Iona", "Brendon" };
+
+//var sortedNames = from name in randomNames
+//                  where name.Length % 2 == 0
+//                  orderby name
+//                  select name;
+
+//Console.WriteLine($"List of names, that are less than zero and less than 10:");
```

Comment:
List of names, that are less than zero and less than 10??
Is this task about it?)